TWLight/resources/helpers.py

```diff
@@ -186,7 +186,7 @@ def _read_translation_file(language_code: str, filename: str):
     if os.path.isfile(filepath):
         with open(filepath, "r") as translation_file:
             translation_dict = json.load(translation_file)
-
+            translation_file.close()
```

Comment:
I think the files that we open in `with` blocks are closed automatically

Reply:
You are correct. I totally read the Django docs incorrectly. **Always read the second sentence**. 🤦‍♀️ 
![Screenshot 2024-05-10 at 11 44 36 AM](https://github.com/WikipediaLibrary/TWLight/assets/30132257/e812d50c-b844-4dc8-8d16-1c5f87e7c769)